packages/cf-style-const/src/variables.js

```diff
@@ -14,6 +14,20 @@ export default {
     seven: 'linear-gradient(left, #F69259, #FFDB6F)',
     eight: 'linear-gradient(left, #85CBA8, #FFDB6F)'
   },
+  color: {
+    rain: '#408BC9',
```

Comment:
Once again opinions and this seem all over the map. 

I like these names but for the sake of time and sanity I'll agree to the first thing other people agree to.

There's 3 ways we can do this:

1. Name them by color. blue1, blue2, blue3 etc or blueDark, blue, blueLight. The advantage here is that there's little mental unpacking on the part of the programmer to figure out what the color names mean. The downside is that it doesn't scale super well if theres a blue lighter than blueLight or blue1.

2. Name them by function. color-primary, color-action-success, etc. We can alias variables as we go along that point at a limited pallette. ex. if button-primary is a blue that gets re-used as a color for info panel borders, we make a new info-panel-border variable that references blue. Advantage is that calling components will be decoupled from actual color names, so renaming is super easy. Downside is there's a lot more variables, and the frequency of new variables adds overhead.

3. Use abstract names like the ones above. Advantage: developers don't think about colors, they just plug in variables indicated in the mocks. There's no naming collisions (like when you have a blue lighter than blueLight or blue1). The downside is that if you want to change all your colors and tomato becomes blue, then you'll need to rename, just like with the first category. Also, if developers really DO want to expend mental energy to decipher the colors, it takes more energy.